dpp4/tests/test_submodel_integration.py

```
@@ -1,8 +1,11 @@
 """Integration tests for submodels with AAS handler."""
 
-import pytest
+import unittest
+from unittest.mock import patch
```

Ruff (0.8.2)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)